TwoFactorAuth/etc/acl.xml

```diff
@@ -12,12 +12,10 @@
             <resource id="Magento_Backend::admin">
                 <resource id="Magento_Backend::system">
                     <resource id="Magento_User::acl">
-                        <resource id="Magento_User::acl_users">
-                            <resource id="Magento_TwoFactorAuth::tfa"
-                                      title="Two Factor Auth"
-                                      translate="title"
-                                      sortOrder="40" />
-                        </resource>
+                        <resource id="Magento_TwoFactorAuth::tfa"
```

Comment:
Since you've spent more time testing the module can you tell me is there a special ACL or custom authorization logic for users resetting/configuring 2FA or making a provider a default one for other users? We can't have users that are allowed making changes to their own 2FA controlling other users' 2FA

Reply:
Before I made any changes there were two permissions:
1) Configuring the global settings for 2fa
2) Configuring personal 2fa

For 1, it was structured fine. For 2, it was nested inside `Magento_User::acl_users` as you can see in this change. Because of how magento permissions work, this means that if a user had the tfa `Magento_User::tfa` permission, they would also have to have the parent permission which allows them to change user settings including reseting credentials, roles, etc which defeats the whole purpose of the rule. If they didn't have the `Magento_User::tfa` role then not only would they not be able to configure personal 2fa but it would actually cause an infinite redirect loop.

Reply:
And what about now? My main concern is users changing other users 2FA settings

Reply:
Now it's up to the admin to configure the acl roles correctly. They can now separately allow tfa/user config/user roles.